app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -281,6 +281,15 @@ class BrowserTabFragment : Fragment(), FindListener, CoroutineScope {
         })
     }
 
+    private fun recoverDaxDialogFromActivity(): Fragment? = activity?.supportFragmentManager?.findFragmentByTag(DAX_DIALOG_DIALOG_TAG)
+
+    private fun removeDaxDialogFromActivity() {
+        val fragment = recoverDaxDialogFromActivity()
+        fragment?.let {
+            activity?.supportFragmentManager?.transaction { remove(it) }
```

Comment:
What's the difference between `remove` or `dialog.dismiss`?

Reply:
dismiss would trigger the listeners and the dialog won't be shown again if the activity gets killed which is not what we are after.